ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -0,0 +1,227 @@
+                ' + Archive Analysis with numbers : {}'.format(analysis_numbers_list))
+            self.analyses_page.search_by_number_and_archive(
+                analysis_numbers_list)
+            self.order_page.get_orders_page()
+            self.order_page.sleep_medium()
+            rows = self.order_page.search(analysis_numbers_list[0])
+            self.order_page.click_check_box(source=rows[0])
+            self.base_selenium.LOGGER.info(
+                ' + archive order has analysis number =  {}'.format(analysis_numbers_list[0]))
+            self.order_page.archive_selected_orders()
+            rows = self.order_page.result_table()
+            self.assertEqual(len(rows), 1)
+
+    def test03_restore(self):
+        """
+        Restore Order
+        I can restore any order successfully
+        LIMS-4374
+        """
+        analysis_numbers = []
+        self.order_page.sleep_medium()
+        self.order_page.get_archived_items()
+        self.order_page.sleep_medium()
+        selected_orders, selected_rows = self.order_page.select_random_multiple_table_rows()
+        for order in selected_rows:
+            analysis_numbers.extend(self.base_selenium.get_row_cell_text_related_to_header(row=order,
+                                                                                           column_value='Analysis No.').split(
+                ','))
+        self.order_page.restore_selected_items()
+        self.order_page.get_active_items()
+        for analysis_number in analysis_numbers:
+            self.assertTrue(self.order_page.is_order_exist(
+                value=analysis_number))
+
+    def test04_deleted_archived_order(self):
+        """
+        New: Order without/with article: Deleting of orders
+        The user can hard delete any archived order
+        LIMS-3257
+        """
+        self.order_page.sleep_medium()
+        self.order_page.get_archived_items()
+        order_row = self.order_page.get_random_order_row()
+        self.order_page.click_check_box(source=order_row)
+        analysis_numbers_list = self.base_selenium.get_row_cell_text_related_to_header(
+            order_row, 'Analysis No.').split(',')
+        self.base_selenium.LOGGER.info(
+            ' + delete order has number = {}'.format(self.base_selenium.get_row_cell_text_related_to_header(
+                order_row, 'Order No.')))
+        self.order_page.delete_selected_item()
+        self.assertFalse(self.order_page.confirm_popup())
+        self.analyses_page.get_analyses_page()
+        self.analyses_page.get_archived_items()
+        self.base_selenium.LOGGER.info(
+            ' + check that analysis numbers {} is deleted successfully'.format(analysis_numbers_list))
+        has_active_analysis = self.analyses_page.search_if_analysis_exist(
+            analysis_numbers_list)
+        self.assertFalse(has_active_analysis, True)
+
+    @skip('https://modeso.atlassian.net/browse/LIMS-4466')
```

Comment:
skip decoration should be before the method not before the parameterized and I can see that this test case passed here.

https://travis-ci.org/modeso-open-source/1lims-automation/jobs/553227755#L160